scripts/gh-pr-assigned.coffee

```diff
@@ -1,9 +1,14 @@
 # Description:
-#   Notifies HUBOT_GITHUB_USERNAME when they have been assigned to a Pull Request
+#   Notifies Github user when they have been assigned to a Pull Request in their chat client of choice
 #
 # Configuration:
-#   HUBOT_GITHUB_EVENT_NOTIFIER_ROOM  - The default room to which message should go (recommend private chat, i.e. your Slack username)
-#   HUBOT_GITHUB_USERNAME - The github user to check for pul request assignment
+#   HUBOT_GITHUB_EVENT_NOTIFIER_ROOMS  - The rooms to which message should go,
+#                                        comma-separated (i.e. your Slack username)
+#   HUBOT_GITHUB_USERNAMES - The github user to check for pull request assignment
+#   (NOTE: rooms and usernames should match in order
+#   i.e. HUBOT_GITHUB_EVENT_NOTIFIER_ROOMS = room1, room2
+#        HUBOT_GITHUB_USERNAMES            = user1, user2
```

Comment:
Not sure how scalable this is:

  HUBOT_GITHUB_EVENT_NOTIFIER_ROOMS = room1, room2, chat4, totallyawesomeplace, slackisdabomb, fullyfunctionalstuff, js-data-awesomeness, <insert 20 more rooms here>
  HUBOT_GITHUB_USERNAMES            = user1, user2, bob, janice, ross, chandler, rachel, phoebe, monica, joey, <insert X more people here>

So given I've configured the above, I'm Phoebe and I want to change my room.... wait, which room am I notifying to?

It might be better to do this as tuplets:

  HUBOT_GITHUB_ROOM_MAPPINGS = user1|room1, user2|room2, joey|sandwichhouse, etc etc

And modify `splitAndSanitiseEnvVars` accordingly:

```coffee
# This will return an array of tuples, so:
#   [[user1, room1], [user2, room2], ...]
splitAndSanitiseEnvVars = (commaSeparatedList) ->
  commaSeparatedList.split(",").map((mapping) ->
    mapping.split(",") -> ((allocation) ->
      [allocation[0], allocation[1]]
    )
  )
```

Does that make sense?